tests/assessment_helpers.py

```diff
@@ -0,0 +1,389 @@
+        }
+
+    def _get_supplier_frameworks(self):
+        return {
+            k: dict(
+                v,
+                supplierId=k,
+                frameworkSlug=self.framework_slug,
+                countersignedPath="",
+                countersignedAt="",
+                declaration=dict(
+                    v["declaration"],
+                    primaryContact=u"Supplier {} Empl\u00f6yee 123".format(k),
+                    primaryContactEmail="supplier.{}.{}@example.com".format(k, self.framework_slug),
+                    mitigatingFactors=u"Permit, brevi manu, Supplier {}\u2019s sight is somewhat troubled".format(k),
+                    mitigatingFactors2=(
+                        u"Supplier {} can scarcely be prepared for every emergency that might crop up".format(k)
+                    ),
+                ),
+            )
+            for k, v in {
+                # this has many fields filled out, the intention being to take a quick dict comprehension-modified copy
+                # blanket-overriding any undesired values where necessary
+                #
+                # onFramework here is set to the expected result for the accompanying declaration, the idea being that
+                # a subclass can override them with mismatched values if necessary
+                1234: {
+                    "onFramework": None,
+                    "declaration": {
+                        "status": "complete",
+                        "shouldBeFalseLax": True,
+                        "shouldBeTrueStrict": True,
+                        "shouldMatchPatternStrict": "HE L Y   S",
+                        "irrelevantQuestion": "Irrelevant answer",
+                    },
+                },
+                2345: {
+                    "onFramework": False,
+                    "declaration": {
+                        "status": "complete",
+                        "shouldBeTrueLax": True,
+                        "shouldMatchPatternLax": "Good pattern",
+                        "shouldBeFalseStrict": None,  # <- subtle but important test here
+                    },
+                },
+                3456: {
+                    "onFramework": False,
+                    "declaration": {
+                        "status": "complete",
+                    },
+                },
+                4321: {
+                    "onFramework": True,
+                    "declaration": {
+                        "status": "complete",
+                        "shouldBeTrueLax": True,
+                        "shouldMatchPatternLax": "Good   Pattern",
+                        "shouldMatchPatternStrict": "H.E. L.  Y",
+                        "irrelevantStupidQuestion": "Irrelevant stupid answer",
+                    },
+                },
+                4567: {
+                    "onFramework": False,
+                    "declaration": {
+                        "status": "bored-gave-up",
```

Comment:
yeah i think this was originally that but i think i noticed that nothing in this area knows/cares about `started` so i thought i'd throw some chaos into the mix